src/Components/Body/Body.js

```diff
@@ -1,13 +1,19 @@
 import React, { Component } from 'react';
-import styles from './Body.module.css';
+import styles from './body.module.css';
 import globalStyles from '../../CSS/global.css';
-import WelcomePop from '../Popup/WelcomePop/WelcomePop';
+import PopupWelcome from '../PopupWelcome/PopupWelcome';
```

Comment:
A good practise here is to create `/PopupWelcome/index.js` with the following content:
```
export { PopupWelcome } from './PopupWelcome`
```

So now you can use it like `import PopupWelcome from '../PopupWelcome'`

here is how node resolving system works

Reply:
Done
33c2ec2